### PR TITLE
HAWQ-683. Updates required for Javadocs api generation

### PR DESCRIPTION
**Pull request by shivzone**
Change for javadoc prior to hosting on hawq apache site.
Package info sites will be updated to add more details in the future.

### CODE REVIEW

File: pxf/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Package Contents
 ================
 
 PXF is distributed as a set of RPMs -
-
+git dif
```
Comment (hornn) [inline pxf/README.md]: dif?

Comment (shivzone) [inline pxf/README.md]: my bad. removed it

**Comment by GodenYao** [comment]
+1

File: pxf/pxf-hdfs/src/main/java/org/apache/hawq/pxf/plugins/hdfs/AvroResolver.java [hunk 2]
```diff
@@ -152,7 +152,7 @@ boolean isAvroFile() {
      * The record can arrive from one out of two different sources: a sequence
      * file or an AVRO file. If it comes from an AVRO file, then it was already
      * obtained as a {@link GenericRecord} when when it was fetched from the
```
Comment (kavinderd) [inline pxf/pxf-hdfs/src/main/java/org/apache/hawq/pxf/plugins/hdfs/AvroResolver.java]: Not related to your change but typo: "when when" should just be "when"

**Comment by kavinderd** [comment]
Looks good

**Comment by sansanichfb** [comment]
Might be useful to have @version tag in each file, but do not add it manually to each class, but just substituting it on some build phase, reading current version from graddle properties, the similar way we do for PXF api version.

**Pull request closed without merging**